### PR TITLE
Add IPFS chart

### PR DESCRIPTION
**Pull request by kmoore134**
@sonicaj - Fixed this up, seems to validate and work properly with dynamic forms now.

### CODE REVIEW

**Review by sonicaj — COMMENTED** [review]
It looks really good, however can we please have this under `test` train ? I'll be updating `charts` train today or tomorrow hopefully as soon as we have the dynamic charts PR finalized, and this can be moved there. Reasoning is that current UI is not able to handle the creation form for this chart and users on 20.12 or master won't be able to install this.

File: test/ipfs/1.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+|:-------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------|
+| `image.repository`                               | Image repository                                                                                                                        | `ipfs/go-ipfs`                    |
+| `image.tag`                                      | IPFS image tag. Possible values listed [here](https://hub.docker.com/r/ipfs/go-ipfs/tags).                                              | `v0.8.0-rc1`   |
+| `image.pullPolicy`                               | Image pull policy                                                                                                                       | `IfNotPresent`                   |
+| `extraArgs`                                      | Additional command line arguments to pass to the IPFS server                                                                            | `[]`                             |
```
Comment (sonicaj) [inline test/ipfs/1.0.0/README.md:22]: We are not using `extraArgs` ?

Comment (sonicaj) [reply test/ipfs/1.0.0/README.md:22]: Also this is not really required plus it has a maintenance overhead as we would have to keep questions.yaml and this table in sync. We can probably have this updated by CI or something similar down the road and remove it even for now ?

File: test/ipfs/1.0.0/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
+{{ $scheme := "http" }}
```
Comment (sonicaj) [inline test/ipfs/1.0.0/templates/deployment.yaml:1]: Is this required ?

File: test/ipfs/1.0.0/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{{ $scheme := "http" }}
```
Comment (sonicaj) [inline test/ipfs/1.0.0/templates/service.yaml:1]: Is this required ?

File: test/ipfs/1.0.0/templates/serviceaccount.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "ipfs.serviceAccountName" . | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ template "ipfs.name" . }}
+    chart: {{ template "ipfs.chart" . }}
+    release: "{{ .Release.Name }}"
```
Comment (sonicaj) [inline test/ipfs/1.0.0/templates/serviceaccount.yaml:9]: `{{ .Release.Name | quote }}` should be better in this case I think

**Review by sonicaj — COMMENTED** [review]
LGTM, just a few minor cosmetic nits

**Review by sonicaj — APPROVED** [review]
Actually I think i can clean up the above bits for all the charts we have as we would have these concerns there as well. Thank you :)

**Comment by kmoore134** [comment]
> Actually I think i can clean up the above bits for all the charts we have as we would have these concerns there as well. Thank you :)

Even better, since I copied this from minio chart, and it'll be good to get them all into proper shape at same time.

Thanks @sonicaj !

**Pull request merged**